multimodal/src/autogluon/multimodal/learners/learner.py

```diff
@@ -0,0 +1,15 @@
+class MultiModalLearner:
```

Comment:
Rename the file as `base.py` and use the following:


```python

class BaseLearner(ABC):
    @abstractmethod
    def fit():
        pass

    @abstractmethod
    def predict():
        pass

    @abstractmethod
    def evaluate():
        pass

    @abstractmethod
    def predict_proba():
        pass

    @abstractmethod
    def extract_embedding():
        pass

    @abstractmethod
    def save():
        pass
    
    @abstractmethod
    def load():
        pass

```

Reply:
okay got it! I actually thought about using ABC but didn't eventually push it through lol. Thanks for the suggestion!

Reply:
@suzhoum uses `abstract` as the keyword in fusion model. Do we need to align the keyword (`abstract` or `base`)? See https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/models/fusion/base.py#L12.